apps/flame-defi/app/pool/types.ts

```diff
@@ -82,26 +108,37 @@ export type PoolContextProps = {
   setModalOpen: (modalOpen: boolean) => void;
   txnStatus: TXN_STATUS;
   setTxnStatus: (txnStatus: TXN_STATUS) => void;
+  maxPrice: string;
+  updateMaxPrice: (
+    feeTier: number,
+    token0Decimals: number,
+    token1Decimals: number,
+  ) => void;
 };
 
 export type PoolPositionContextProps = {
   feeTier: string;
-  symbols: string[];
-  selectedSymbol: string;
+  rawFeeTier: number; // The unformatted fee tier value to be used in calculations
+  selectedSymbol: string; // The symbol of the token that is currently in the ToggleSwitch. Controls which set of token liquidity and price data is displayed
   handleReverseTokenData: (symbol: string) => void;
-  collectAsNative: boolean;
+  collectAsNative: boolean; // This boolean controls the toggle in the UI to collect fees as the native token
   handleCollectAsNative: (collectAsNative: boolean) => void;
-  poolTokenOne: PoolToken | null;
-  poolTokenTwo: PoolToken | null;
+  poolToken0: PoolToken | null;
+  poolToken1: PoolToken | null;
+  poolPosition: PoolPositionResponse | null;
   currentPrice: string;
   minPrice: string;
   maxPrice: string;
+  isReversedPoolTokens: boolean; // This boolean is only used by the pool position details page and the pool context to reverse the token order based on the selected symbol
+  isPositionClosed: boolean; // This boolean is use to know when the position is closed and to hide collect fees and remove liquidity buttons
+  refreshPoolPosition: () => void;
+  positionNftId: string;
 };
 
 export type PoolTxnStepsProps = {
   txnStatus: TXN_STATUS;
   poolTokens: PoolToken[];
-  txnHash: string;
+  txnHash: HexString | undefined;
```

Comment:
```suggestion
  txnHash?: HexString;
```

Reply:
There won't be a transaction hash yet when the first modal is opened